src/modules/core/components/Button/Button.tsx

```diff
@@ -22,7 +22,8 @@ export interface Appearance {
     | 'blue'
     | 'no-style'
     | 'white'
-    | 'dottedArea';
+    | 'dottedArea'
+    | 'pink';
```

Comment:
Why was this added? This already exists!!! It's called `danger`

Please remove the "pink" theme

Reply:
It doesn't exist

`themeDanger` uses:   --danger: rgb(248, 43, 101);
`themePink` uses:   --pink: rgb(254, 94, 124);

The pink looks kind of red, but it's a noticeable different shade of red (pinkish red :D) than the one that uses the danger theme. Should I still remove it?

Reply:
Yes, remove it. We can deal with polishing the app at a later date.